enterprise-suite/console-api/prometheus.yml

```diff
@@ -48,6 +48,8 @@ scrape_configs:
         replacement: /api/v1/nodes/${1}/proxy/metrics/cadvisor
       - source_labels: [__meta_kubernetes_node_name]
         target_label: node_name
+      - source_labels: [__meta_kubernetes_node_name]
+        target_label: node
```

Comment:
I notice that you're duplicating a tag here, and leaving `node_name` across the various data sets. Is there a reason to do it this way instead of, say, rename the kube_state_metrics `node` tag to `node_name`?  It looks like that would make everything consistent with no dupes or holes

Reply:
Since this is a primary key for joining across data sets, you could add a consistency test. Should be easy, and would catch breakage of this in future.